tests/array.rs

```diff
@@ -709,6 +709,19 @@ fn test_select() {
     assert_abs_diff_eq!(c, c_target);
 }
 
+#[test]
+fn test_select_1d() {
+    let x = arr1(&[0, 1, 2, 3, 4, 5, 6]);
+    let r1 = x.select(Axis(0), &[1, 3, 5]);
```

Comment:
It would be nice to have a repeated index and an index which is not in increasing order.